configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
I think I have figured out the correct solution.
I plan to add wxUSE_IMAGES_IN_RESOURCES to select whether wxICON looks for xpm or resource file under windows. The main code changes will be in "wx/gdicmn.h"; but, I will have to change most places that check wxHAS_IMAGES_IN_RESOURCES to instead check wxUSE_IMAGES_IN_RESOURCES. So, nearly every sample will need changed and a few demos and utils. Tim S.